doc/source/cluster/cloud.rst

```diff
@@ -0,0 +1,225 @@
+.. _cluster-cloud:
+
+Launching clusters on the cloud
+===============================
+
+This section provides instructions for configuring the Ray Cluster Launcher to use with AWS/Azure/GCP, an existing Kubernetes cluster, or on a private cluster of host machines.
```

Comment:
I would recommend linking to my [blog post here](https://medium.com/distributed-computing-with-ray/a-step-by-step-guide-to-scaling-your-first-python-application-in-the-cloud-8761fe331ef1)

In general, it's a great way to provide links for people to discover other content.